ruby/cross_service_examples/feedback_sentiment_analyzer/comprehend_lambda_handler.rb

```diff
@@ -2,19 +2,19 @@
 # SPDX-License-Identifier: Apache-2.0
 # frozen_string_literal: true
 
-require "json"
-require "aws-sdk-comprehend"
-require "logger"
+require 'json'
+require 'aws-sdk-comprehend'
+require 'logger'
 
 def lambda_handler(event:, context:)
   logger = Logger.new($stdout)
 
   logger.info("event:\n #{event}\n")
   logger.info("context:\n #{context}\n")
 
-  rekognition_client = Aws::Comprehend::Client.new(region: event["region"])
+  rekognition_client = Aws::Comprehend::Client.new(region: event['region'])
```

Comment:
named rek_client but looks like Comprehend? Can you look here again.

Reply:
Fixed.